Examples/Sources/ChatbotExample/Views/ChatSettingsDialog.swift

```diff
@@ -0,0 +1,466 @@
+                        selectedModel: $model.selectedModel
+                    )
+                }
+            } else {
+                VStack(spacing: 8) {
+                    ForEach(model.availableModels) { modelInfo in
+                        DynamicLLMOptionButton(
+                            modelInfo: modelInfo,
+                            selectedModel: $model.selectedModel
+                        )
+                    }
+                }
+            }
+        }
+    }
+    
+    // MARK: - Temperature Section
+    
+    private var temperatureSection: some View {
+        VStack(alignment: .leading, spacing: 12) {
+            Text("Temperature")
+                .font(.headline)
+                .fontWeight(.semibold)
+            
+            VStack(spacing: 8) {
+                HStack {
+                    Text("More focused")
+                        .font(.caption)
+                        .foregroundColor(.gray)
+                    
+                    Spacer()
+                    
+                    Text("\(String(format: "%.1f", model.temperature))")
+                        .font(.caption)
+                        .fontWeight(.medium)
+                        .foregroundColor(AppColors.primary)
+                        .padding(4)
+                    
+                    Spacer()
+                    
+                    Text("More creative")
+                        .font(.caption)
+                        .foregroundColor(.gray)
+                }
+                
+                // Temperature slider
+                Slider($model.temperature, minimum: 0.0, maximum: 1.0)
+            }
+        }
+    }
+    
+    // MARK: - Max Tokens Section
+    
+    private var maxTokensSection: some View {
+        VStack(alignment: .leading, spacing: 12) {
+            Text("Max Tokens")
+                .font(.headline)
+                .fontWeight(.semibold)
+            
+            TextField("1000", text: $model.maxTokens)
+                .foregroundColor(.black)
+                .padding(12)
+                .background(Color.gray.opacity(0.1))
+                .cornerRadius(8)
+        }
+    }
+    
+    // MARK: - Footer Section
+    
+    private var footerSection: some View {
+        HStack(spacing: 16) {
+            Button("❌ ") {
+                isPresented = false
+            }
+            .foregroundColor(.black)
+            .secondaryButtonStyle()
+            .frame(maxWidth: .infinity)
+            
+            Button("💾 ") {
+                model.saveSettings()
+                onSave()
+                isPresented = false
+            }
+            .foregroundColor(.white)
+            .secondaryButtonStyle()
+            .frame(maxWidth: .infinity)
+        }
+        .padding(20)
+        .background(Color.gray.opacity(0.2))
+        .frame(maxWidth: .infinity)
+    }
+}
+
+// MARK: - Model Option Button
+
+struct LLMOptionButton: View {
```

Comment:
This seems almost identical to DynamicLLMOptionButton. Combine the two views into one.